src/clib/lib/include/ert/enkf/enkf_types.hpp

```diff
@@ -61,10 +61,10 @@ enkf_var_type;
 */
 
 typedef enum {
-    FIELD = 104,    /* WELL has been removed  */
-    GEN_KW = 107,   /* RELPERM has been removed & HAVANA_FAULT */
-    SUMMARY = 110,  /* TPGZONE has been removed */
-    GEN_DATA = 113, /* PILOT_POINT has been removed */
```

Comment:
noooo where did the meaningful and informative comments go Eivind how could you :shocked pickachu: